src/main/scala/akka/contrib/process/NonBlockingProcess.scala

```diff
@@ -183,6 +183,8 @@ class NonBlockingProcess(
   private val inspectionTick =
     context.system.scheduler.schedule(inspectionInterval, inspectionInterval, self, Inspect)
 
+  private implicit val materializer = ActorMaterializer()(context.system)
+
```

Comment:
We need to be careful about the materialiser here. The placement of the one before was careful given the use of context.system - which is unusual. I'd prefer to revert this but.